### ISSUE TITLE
Carla leaderboard 2.0 release for windows OS?

### ISSUE THREAD

**Issue by YangyangFu** [carla-simulator/carla#6883]
Following the setup package for leaderboard 2.0 [here](https://leaderboard.carla.org/get_started/), I realized that the leaderboard 2.0 requires a specific version of Carla, and the link for the Carla release seems only for Ubuntu/Linux OS. I'm running Carla server on windows OS now. So is there any release for windows OS?

**Comment by Turuu1124** [comment]
same question did you find out?

**Comment by YangyangFu** [comment]
No. My understanding is we need rebuild the specific release locally for Windows. I didn't try it though. 

I simply contained their ubuntu release into a docker file, and run the docker on my Windows.
